.codegen/exception.java.tmpl

```diff
@@ -0,0 +1,24 @@
+// Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+package com.databricks.sdk.core.error;
+
+import com.databricks.sdk.core.DatabricksError;
+import com.databricks.sdk.support.Generated;
+
+import java.util.List;
+
+/**
+ {{.Comment " * " 80}}
+ */
+@Generated
+public class {{.PascalName}} extends {{if .Inherit -}}{{.Inherit.PascalName}}{{else}}DatabricksError{{end}} {
+  public {{.PascalName}}(String message, List<ErrorDetail> details) {
+    super("{{.Name}}", message, {{.StatusCode}}, details);
+  }
+
+  {{if not .Inherit}}
+  {{.PascalName}}(String errorCode, String message, int statusCode, List<ErrorDetail> details) {
+      super(errorCode, message, statusCode, details);
+  }
+  {{end}}
```

Comment:
This case isn't used yet, but I suspect you may want to use it if you follow my comments in AbstractErrorMapper.java.